### DISCUSSION TITLE
Find three 11-letter words

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#930] [Galaxy Trivia] [answered]
Which use the letters from `ANTHROPOLOGIST`.

**Comment by tomerf** [comment]
rhinoplasty
throatiness
anorthosite

**Reply by ParanoidUser** [reply]
Nice try! Only letters from "anthropology" should be used in new words. 😊

**Comment by YashNuhash** [comment]
Here are the three words below:

- Protoplasts
- Pathologist
- Parasitology

**Reply by ParanoidUser** [reply]
Nice try! Only letters from "anthropology" should be used in new words. 😊

**Comment by peterbaccaro** [comment, marked as answer]
I have found four 11-letter words, they are:

1. ANTHOLOGIST
2. PATHOLOGIST
3. PHONOLOGIST
4. PROTAGONIST
